pages/about/groups/agwg.md

```diff
@@ -33,67 +34,79 @@ Some information on this page is also shown on, and may be more current in, the
 {% include_cached toc.html type="end" %}
 {:/}
 
-## Announcements
+## Charter
 
-No announcements at the moment.
+The group is chartered until October 2025.
 
-## Current Work
+To learn about the group’s current focus, scope and deliverables, see the [AG Working Group Charter](https://www.w3.org/WAI/GL/charter).
 
-The Working Group maintains a comprehensive list of [publications and current timelines](https://www.w3.org/WAI/GL/wiki/Timelines).
+## Current work
 
-The [Project Plan](https://www.w3.org/WAI/GL/wiki/Timelines) details intended timeline and milestones for this work. A [list of publications](https://www.w3.org/TR/tr-groups-all#tr_Web_Content_Accessibility_Guidelines_Working_Group) on the W3C Technical Reports page includes completed deliverables that are no longer worked on by the Working Group.
+The work of the AG Working Group follows the [W3C Process](https://www.w3.org/2023/Process-20231103/). 
+
+This work is partly funded by the [WAI Core 2015 Project](https://www.w3.org/WAI/about/projects/wai-core-2015/). The work of the AG Working Group does not necessarily reflect the views or policies of the funders.
```

Comment:
Again this seems to have crept in, although I think in this case it is needed. The trouble is where to put it. Not really in 'Current work'. Not sure it fits anywhere in new structure. Closest might be 'Contribute to the work'. Open to ideas.

Reply:
I did think the information about who funds the work sits best under 'current work, since it's related. 
But if that doesn't make sense, then how about we place it under 'Charter and funding'?

Reply:
No, I think it needs to be separate from the Charter

Reply:
We could place the information under a new h2 heading called 'About the work'. Under that heading will be two h3 headings:
  - Current work
  - Funding
I've mocked this up on the ARIA WG page that I committed this morning.

Or we could place the information under its own heading, 'Funding', after 'Publications and copyright'?

I don't think the funding information sits logically under 'Contribute to the work' because this is just information about the group (how it is funded) and not a call to action (i.e. to contribute funding), like the other items in this section.

Reply:
I don't think it should be close to the 'Current work'. Even having a 'funding' section doesn't make sense to me. If you take it out for the moment and then we can discuss in New Year - we could publish without it and then fix following discussion.

Reply:
Will do. This information was removed by a reviewer in the original mock up of the proposed design, so perhaps the information isn't needed on this page and is more appropriate on the 'Sponsor WAI page'?